### PR TITLE
Added Verse token

### PR DESCRIPTION
**Pull request by just-software-dev**
Added Verse token

https://etherscan.io/token/0x249ca82617ec3dfb2589c4c17ab7ec9765350a18

### CODE REVIEW

File: assets/general/verse/info.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "name": "VERSE",
```
Comment (adamant-al) [inline assets/general/verse/info.json:2]:
```suggestion
  "name": "Verse",
```

**Review by adamant-al — CHANGES_REQUESTED** [review]
Also

```
  "defaultVisibility": true,
  "defaultOrdinalLevel": 99,
```

See https://github.com/Adamant-im/adamant-wallets/pull/40/files

**Comment by just-software-dev** [comment]
iOS tested, everything works

**Review by adamant-al — APPROVED** [review]

File: assets/blockchains/ethereum/verse/info.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+  "name": "Verse",
+  "symbol": "VERSE",
+  "status": "active",
+  "defaultVisibility": true,
+  "defaultOrdinalLevel": 99,
```
[the remark this answers is not in the document]
Comment (adamant-al) [reply assets/blockchains/ethereum/verse/info.json:6]:
```suggestion
  "defaultOrdinalLevel": 95,
```

**Review by bludnic — APPROVED** [review]

File: assets/general/verse/info.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  "status": "active",
+  "createCoin": false,
+  "links": [
+    {
+      "name": "twitter",
+      "url": "https://twitter.com/BitcoinCom"
+    },
+    {
+      "name": "medium",
+      "url": "https://blog.bitcoin.com/"
```
Comment (confleux) [inline assets/general/verse/info.json:18]:
```suggestion
      "url": "https://blog.bitcoin.com"
```

**Comment by adamant-al** [comment]
@just-software-dev Merge it.

**Pull request merged**